aux_assistant.py

```diff
@@ -260,7 +262,7 @@ def validate_login():
     '''Validates login'''
     user = str(request.form.get("UserName"))
     if not user in USERSET:
-        flash('Username and/or Passeword invalid, try again or click below to Sign Up')
+        flash('Username and/or Password invalid, try again or click below to Sign Up')
         return redirect(url_for('login_page'))
     global USER
```

Comment:
With flask login being set up now, you can probably delete this global USER variable since that was just for testing.